data/json/recipes/other/tool.json

```diff
@@ -1122,5 +1122,19 @@
     "qualities": [ { "id": "HAMMER", "level": 1 }, { "id": "CUT", "level": 1 } ],
     "tools": [ [ [ "surface_heat", 10, "LIST" ] ] ],
     "components": [ [ [ "plastic_chunk", 5 ] ], [ [ "2x4", 1 ], [ "stick", 1 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "chain_wrench",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
+    "difficulty": 2,
+    "time": "1 h",
+    "reversible": true,
+    "autolearn": true,
+    "skills_required": [ "mechanics", 2 ],
+    "qualities": [ { "id": "SCREW", "level": 1 }, { "id": "SAW_M", "level": 2 }, { "id": "DRILL", "level": 3 } ],
+    "components": [ [ [ "scrap", 20 ] ], [ [ "pipe", 1 ] ], [ [ "chain", 1 ] ] ]
```

Comment:
This recipes does not produce any of these:
![image](https://user-images.githubusercontent.com/42699974/88619444-6b0a0000-d050-11ea-8e31-aaf1aa1ef5f9.png)
![image](https://user-images.githubusercontent.com/42699974/88619454-6e9d8700-d050-11ea-9bf6-0165458bc16b.png)
![image](https://user-images.githubusercontent.com/42699974/88619468-74936800-d050-11ea-84c1-d2d674f629cf.png)
![image](https://user-images.githubusercontent.com/42699974/88619474-765d2b80-d050-11ea-8e6a-6a75430ceb64.png)